.vsts-dotnet-ci.yml

```diff
@@ -179,29 +179,29 @@ jobs:
     pool:
       name: NetCore-Public
       demands: ImageOverride -equals 1es-windows-2019-open
-    helixQueue: Windows.10.Amd64.Open
+    helixQueue: Windows.11.Amd64.Client.Open
```

Comment:
Alternatively we could use `Windows.Amd64.Server2022.Open`, or `Windows.Amd64.Server2025.Open`